doc/tutorials/dune-package-management/setup.md

```diff
@@ -0,0 +1,197 @@
+# Setting up Package Management with Dune
+
+The idea of package management with Dune has been to be as unobtrusive as
+possible. Thus most projects can easily be built with just the minimum of
+changes.
+
+In this tutorial we'll create a simple project to use the OCaml package
+management for the very first time.
+
+## Declare dependencies
+
+The best way to work with the package management is to declare your
```

Comment:
Maybe a link to how to declare opam file in Dune could add some value here?

Reply:
Good point, I originally wanted to add it but couldn't figure out the syntax. But after asking @emillon how to do it, it's linked now :+1: